understanding/reflow.html

```diff
@@ -58,284 +56,193 @@
                   
-                  <p>How much of the content is visible may change at different scales. For example, navigation
-                     menus that are fully visible in the desktop layout are often collapsed into fewer
-                     items, or even into a single menu button (the 'hamburger' icon pattern) so they take
-                     up less screen space.
+                  <p>どの程度の量のコンテンツが表示されるかは、縮尺によって異なる。例えば、デスクトップレイアウトで完全に表示されるナビゲーションメニューは、画面のスペースが少なくて済むため、少数の項目、又は 1 つのメニューボタン (「ハンバーガー」アイコンのパターン) にまとめられて表示されることがある。
                   </p>
                   
-                  <p>The Success Criterion is met as long as all content and functionality are still fully
-                     available -  either directly, or revealed via accessible controls, or accessible via
-                     direct links.
+                  <p>全てのコンテンツと機能が直接、又はアクセス可能なコントロールを介して、又は直接リンクのいずれかを介して完全に利用可能である限り、達成基準は満たされる。
                   </p>
                   	
                </section>
                <section id="content-exceptions-for-reflow">
                   
                   
-                  <h3>Content exceptions for reflow</h3>
+                  <h3>リフローにおけるコンテンツの例外</h3>
                   
-                  <p>Content which requires two-dimensional layout for usage or meaning cannot reflow without
-                     loss of meaning, and is therefore excepted from the need to be presented without two-dimensional
-                     scrolling. For example, graphics and video are by their nature two-dimensional. Cutting
-                     up an image and stacking the blocks would render the content unusable. However, it
-                     <em>is</em> possible to have these elements stay within the bounds of viewport even as other
-                     content zooms to 400% (see advisory techniques).
+                  <p>利用や意味の理解のために 2 次元レイアウトを必須とするコンテンツは、意味を失うことなくリフローすることはできない。したがって、 2 次元スクロールなしで表示する必要性から除外される。例えば、画像及び動画は性質上 2 次元である。画像を切り取って縦に並べるとコンテンツは使用できなくなる。ただし、他のコンテンツが 400 ％に拡大されても、これらの要素をビューポートの境界内に収めたままにすることはできる (参考達成方法を参照)。
                   </p>
                   
-                  <p>Data tables have a two-dimensional relationship between the headings and data cells.
-                     This relationship is essential to convey the content. This Success Criterion therefore
-                     exempts data tables from needing to display without scrolling in the direction of
-                     text (e.g., horizontally in a vertically scrolling page). However, cells within data
-                     tables are not excepted unless the cell contains types of content that also requires
-                     two-dimensional layout for usage or meaning.
+                  <p>データテーブルは、見出しとデータセルの間に 2 次元の関係がある。この関係性はコンテンツの意味を伝えるために不可欠である。したがって、この達成基準では、データテーブルをテキストの方向 (例えば、縦にスクロールするページであれば横方向) にスクロールすることなく表示させる必要はない。ただし、データテーブル内のセルは、用法又は意味のために 2 次元レイアウトを必要とする種類のコンテンツを含まない限り、除外されない。
```

Comment:
- 「関係がある。この関係性は、」は微妙に呼応してない？ような気も…？
- essentialは一応WCAG用語集にあわせておきます
- exemptを辞書に寄せつつ、語順をちょっと入れ替えてみます
- 「利用又は意味」とこのセクションの冒頭で言っているのであわせてみます

```suggestion
                  <p>データテーブルは、見出しとデータセルの間に 2 次元の関係性がある。この関係性はコンテンツの意味を伝えるために必要不可欠である。したがって、この達成基準では、テキストの方向にスクロールせずに表示する必要からデータテーブルを除外する(例えば、縦にスクロールするページであれば横方向)。ただし、データテーブル内のセルは、利用又は意味のために 2 次元レイアウトを必要とする種類のコンテンツを含まない限り、除外されない。
```

Reply:
だんだんよくわからなくなってきましたね…。
「スクロールせずに表示する必要から除外する」というのが二重否定のようになっていて、最後の文の「含まない限り、除外されない。」というのも肯定なのか否定なのか…。

Reply:
すみません、ちょっとこねくり回しすぎた気がします...。

基本的にはもんどさんの修正提案の通りでよいと思いますが、一点だけ、

> したがって、この達成基準では、テキストの方向にスクロールせずに表示する**必要から**データテーブルを除外する(例えば、縦にスクロールするページであれば横方向)。
> This Success Criterion therefore exempts data tables **from needing** to display without scrolling in the direction of text (e.g., horizontally in a vertically scrolling page).

という表現の、「〜する必要から」が (前提知識を持たない読者が読んだときに)「〜する必要があるので」の意味に誤読して文意が捉えにくくなるかも、と思っただけでして。

> したがって、この達成基準では、テキストの方向にスクロールせずに表示する必要**性**からデータテーブルを除外する(例えば、縦にスクロールするページであれば横方向)。

...という具合に、「必要」を「必要性」にする形で誤読を防ぐ、というのもありかなと思いました。

Reply:
自分で言っておいてなんですが、「したがって」の文は、一周回って土屋さんの最初の訳でいいのでは…？となってきましたね…（汗）最後の文はunlessを単に「でない場合」と取ったほうがよいかも？
というのを踏まえて、下記ですと、達成基準の例外になるかどうかという話がすっきりしますかね…？（ちょっと意訳気味ですが）

データテーブルは、見出しとデータセルの間に 2 次元の関係性がある。この関係性はコンテンツの意味を伝えるために必要不可欠である。したがって、この達成基準では、データテーブルをテキストの方向 (例えば、縦にスクロールするページであれば横方向) にスクロールせずに表示させるのを必要としない。ただし、データテーブル内のセルが、利用又は意味のために 2 次元レイアウトを必要とする種類のコンテンツを含まない場合は、例外とはならない。

Reply:
@momdo おお、なるほどです。exempt を意訳気味にした感じですね。でも、このほうが文意としては、伝わりやすい感じがします。ありがとうございます！